docs/source/download.rst

```diff
@@ -1,64 +1,94 @@
-=============
-Obtain Devito
-=============
-
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_.
+For those looking to use Devito alongside other packages as part of a project, we support
+`pip installation`_. If you are looking to develop for Devito, you might benefit from
+following the `conda installation`_  that includes all the bells and whistles we
 recommend when developing for Devito.
 
+
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+Docker for CPUs
+```````````````
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
```

Comment:
for some steps. Prepend `sudo` to the commands if necessary.

Reply:
tagging @FabioLuporini. May not always be necessary.

Reply:
that doesn't make sense to me, if you need sudo you're already defeating _some_ of the purpose of Docker...

if your user isn't in a docker group, ok that's an issue, and that's the only reason you may need to use sudo, but we're not looking good if we expected people to be able to run as sudo

Reply:
TensorFlow does not prepend sudo to their commands and instead links to this [article](https://docs.docker.com/engine/install/linux-postinstall/).